### PR TITLE
Add 4.11 branch to cluster-group-upgrades-operator

### PR DESCRIPTION
**Pull request by rcarrillocruz**

### CODE REVIEW

**Comment by rcarrillocruz** [comment]
@nishant-parekh @vitus133 @danielmellado

**Comment by rcarrillocruz** [comment]
debugging...

File: ci-operator/config/openshift-kni/cluster-group-upgrades-operator/openshift-kni-cluster-group-upgrades-operator-release-4.11.yaml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+- dockerfile_path: Dockerfile.recovery
+  from: base
+  to: cluster-group-upgrades-operator-recovery
+operator:
+  bundles:
+  - dockerfile_path: bundle.Dockerfile
+  substitutions:
+  - pullspec: quay.io/openshift-kni/cluster-group-upgrades-operator:.*
+    with: pipeline:cluster-group-upgrades-operator
+promotion:
```
Comment (danielmellado) [inline ci-operator/config/openshift-kni/cluster-group-upgrades-operator/openshift-kni-cluster-group-upgrades-operator-release-4.11.yaml:29]: missing `disabled: true` under promotion config

Comment (danielmellado) [reply ci-operator/config/openshift-kni/cluster-group-upgrades-operator/openshift-kni-cluster-group-upgrades-operator-release-4.11.yaml:29]: After that, this should work and match the different openshift configs for the release.

**Comment by nishant-parekh** [comment]
@rcarrillocruz I think this branch needs to be rebased

**Comment by rcarrillocruz** [comment]
@nishant-parekh i will once 4.11 branch is opened, this will stay on hold till then

**Comment by rcarrillocruz** [comment]
@nishant-parekh when you get a sec, can you pls review?

**Review by nishant-parekh — APPROVED** [review]

**Pull request merged**